types/react-syntax-highlighter/index.d.ts

```diff
@@ -106,824 +106,824 @@ declare module "react-syntax-highlighter/prism" {
     > {}
 }
 
-declare module "react-syntax-highlighter/dist/styles/hljs" {
+declare module "react-syntax-highlighter/dist/esm/styles/hljs" {
```

Comment:
This seems like a breaking change for anyone still using v10.

Reply:
actually, this was a mistake in v10 types AFAICT, as the changelog of the lib mentions that as a breaking change between 9 and 10.

and this is missing some of the updates, as languages also moved.